src/core/keychain/RainbowSigner.ts

```diff
@@ -63,9 +64,10 @@ export class RainbowSigner extends Signer {
       value: transaction.value
         ? BigNumber.from(transaction.value).toHexString()
         : undefined,
-      nonce: transaction.nonce
-        ? BigNumber.from(transaction.nonce).toHexString()
-        : undefined,
+      nonce:
+        transaction.nonce !== undefined
```

Comment:
out of curiosity, was this a bug?

Reply:
for the case nonce is 0

Reply:
i sent the first tx in a custom network and we weren't sending it because of this